### PR TITLE
Update app log message for notification URL

### PR DESCRIPTION
**Pull request by santhoshb-msft**
Update app log message for notification URL

### CODE REVIEW

File: src/Services/Helpers/UrlValidator.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+namespace Marketplace.SaaS.Accelerator.Services.Helpers;
+
+/// <summary>
+/// URL Validator.
+/// </summary>
+public class UrlValidator
+{
+    /// <summary>
+    /// Validates the URLfor HTTPS.
```
Comment (007zszmz) [inline src/Services/Helpers/UrlValidator.cs:11]: missing a white space.

File: src/Services/Helpers/UrlValidator.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    /// Helps validate if the URL is a valid HTTPS URL.
+    /// </summary>
+    public static bool IsValidUrlHttps(string url)
+    {
+        return Uri.TryCreate(url, UriKind.Absolute, out Uri uriResult) && 
+               uriResult.Scheme == Uri.UriSchemeHttps && 
+               uriResult.Port == 443;
```
Comment (007zszmz) [inline src/Services/Helpers/UrlValidator.cs:18]: how about if we test locally with a different port number than 443?

Comment (santhoshb-msft) [reply src/Services/Helpers/UrlValidator.cs:18]: @007zszmz I think the idea is we dont them to use anything other than https. we can disable this while testing if we need to. Happy to chat internally

**Review by msalsouri — COMMENTED** [review]
review

**Review by 007zszmz — APPROVED** [review]

File: src/AdminSite/Controllers/ApplicationConfigController.cs [hunk 3]
```diff
@@ -119,6 +116,13 @@ public IActionResult ApplicationConfigDetails(int Id)
     public IActionResult ApplicationConfigDetails(ApplicationConfiguration appConfig)
     {
         appConfig.Value = appConfig.Value ?? string.Empty;
+        
+        //check with the config is webhnotifcation url then validate if its proper url
+        if (appConfig.Name == "WebNotificationUrl" && !UrlValidator.IsValidUrlHttps(appConfig.Value))
```
[the remark this answers is not in the document]
Comment (santhoshb-msft) [reply src/AdminSite/Controllers/ApplicationConfigController.cs]: updated accordingly.

File: src/Services/Helpers/UrlValidator.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (santhoshb-msft) [reply src/Services/Helpers/UrlValidator.cs]: fixed the same

**Pull request merged**